specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-05-15/examples/KustoClustersCheckNameAvailability.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-123456789098",
+    "api-version": "2019-01-21",
```

Comment:
Looks like the version numbers got reverted to the previous version

Reply:
I fixed it